app/uk/gov/hmrc/hec/testonly/models/SaveTaxCheckRequest.scala

```diff
@@ -30,6 +30,7 @@ final case class SaveTaxCheckRequest(
   verifier: Either[CRN, DateOfBirth],
   expiresAfter: LocalDate,
   createDate: ZonedDateTime,
+  taxCheckStartDateTime: Option[ZonedDateTime],
```

Comment:
might be good to alert the QA team about this change, we have in the past made changes and the QA team only finds out when things break

Reply:
wiill do after changes in FE and BE 👌